.github/workflows/ci.yml

```
@@ -203,6 +203,45 @@ jobs:
           fail-on-alert: true
       - run: node ./dist/scripts/ci_validate_modification.js before_data.js 'Catch2 Benchmark'
 
+  julia-benchmark:
+    name: Run Julia benchmark example
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+      - uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('package-lock.json') }}
+      - run: npm ci
+      - run: npm run build
+      - name: Save previous data.js
+        run: |
+          git fetch origin gh-pages
+          git checkout gh-pages
+          cp ./dev/bench/data.js before_data.js
+          git checkout -
+      - uses: julia-actions/setup-julia@v1
+        with:
+          version: '1'
+          arch: x64
+      - name: Run benchmark
+        run: |
+          cd examples/julia
+          julia --project --color=yes -e '
+            using Pkg;
+            Pkg.instantiate();
+            include("fib.jl")'
+      - name: Store benchmark result
+        uses: ./
+        with:
+          name: Julia benchmark
+          tool: "julia"
+          output-file-path: examples/julia/output.json
+          skip-fetch-gh-pages: true
+          fail-on-alert: true
+      - run: node ./scripts/ci_validate_modification.js before_data.js 'Julia benchmark'
```

The path changed in the PR I've merged today
```suggestion
      - run: node ./dist/scripts/ci_validate_modification.js before_data.js 'Julia benchmark'
```

---

thanks!